test/functional/feature_bind_extra.py

```diff
@@ -45,8 +45,8 @@ def setup_network(self):
 
         loopback_ipv4 = addr_to_hex("127.0.0.1")
 
-        # Start custom ports after p2p and rpc ports.
```

Comment:
The comment above here is now no longer true and could be removed/altered.
https://github.com/bitcoin/bitcoin/blob/fa46ea0196fda9e41a2feeeed47ff4f6b9d7dfab/test/functional/feature_bind_extra.py#L36-L39

`p2p_port` is usable in `set_test_params` since #25312. (arguably that PR should have changed the comment as well)

Reply:
Thx, fixed.